### DISCUSSION TITLE
Why file upload is not available when I host via docker

### DISCUSSION THREAD

**Discussion by lionsheep0724** [reflex-dev/reflex#733] [Q&A]
File upload is not available when I host simple app via docker container. Nothing happens when I click upload button, after selecting an audio file. No error logs reported, also. But the same scenario works perfectly in local environment. The code is almost same with example code, as below. I mounted volume for uploaded audio file. Refer to below simple docker-compose.yml

```
#app.py
async def handle_upload(self, file: pc.UploadFile):
        """Handle the upload of a file.

        Args:
            file: The uploaded file.
        """
        upload_data = await file.read()
        outfile = f"/app/data/{file.filename}"

        # Save the file.
        with open(outfile, "wb") as f:
            f.write(upload_data)

        # Update the audio file path
        self.audio_file_path = outfile

#docker-compose.yml
version: "3.9"
services:
  client:
    container_name: my-demo
    image: my-demo-client:0.0.0
    volumes:
      - './app/data:/app/data'
    ports:
      - "3000:3000"
      - "8000:8000"
```
Any similar issues?

**Comment by masenf** [comment]
Are you running in dev mode or prod mode in the container?
